circleci-postgres-ram/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM circleci/postgres:11.5-alpine-postgis-ram
+FROM cimg/postgres:12.8-postgis
```

Comment:
what about `ram` part?

Reply:
unfortunately they don't have the ram version for the next gen:
https://hub.docker.com/r/cimg/postgres/tags?page=1